src/IO/FileExplorer.h

```diff
@@ -16,13 +15,40 @@ class FileExplorer {
 
     FileExplorer();
 
+    /**
+     * Sets the currently visited directory to currentDir.
+     *
+     * @param currentDir
```

Comment:
Should have a small description of `currentDir`

Reply:
Done.